src/main/java/io/cryostat/targets/AgentClient.java

```diff
@@ -451,7 +462,7 @@ private <T> Uni<HttpResponse<T>> invoke(
         HttpRequest<T> req =
                 webClient
                         .request(mtd, getUri().getPort(), getUri().getHost(), path)
-                        .ssl("https".equals(getUri().getScheme()))
+                        .ssl("https".equals(getUri().getScheme()) && tlsEnabled)
```

Comment:
If TLS is enabled (required) earlier on and the URI scheme is not HTTPS, then we should not have got here anyway. I think it's still sensible for the request to enable TLS (SSL) if the scheme is HTTPS, or use plain HTTP if that's what the URI specifies. The connection is only going to actually work if the scheme is properly matched here anyway.